tests/qa-integration/pmm_pgsm_integration_test.js

```diff
@@ -410,3 +410,43 @@ Scenario(
     await I.verifyCommand(`docker exec ${container_name} pmm-admin list | grep "postgresql_pgstatmonitor_agent" | grep "Waiting"`);
   },
 );
+
+Scenario(
+  'PMM-T1292 PMM-T1302 PMM-T1303 PMM-T1283 Verify that pmm-admin inventory add agent postgres-exporter with --log-level flag adds PostgreSQL exporter with corresponding log-level @not-ui-pipeline @pgsm-pmm-integration',
+  async ({
```

Comment:
don't this and the following tests fit the parametrized concept?
looks like all except the final check `await inventoryAPI.verifyAgentLogLevel('postgresql', dbDetails);` may be moved out of the test into `BeforeSuite` and then data provider applied?

Reply:
Not really, it can fit into data provider only when all scenarios using same setup, but this is specific to only this scenario + if I use data provider, I wll have to wait for agent metrics hitting for each so instead of waiting only for 30 seconds once I will have to wait for 30 seconds * multiple of scenarios.